### ISSUE TITLE
Are the following class added to the maskrcnn benchmark / data / datasets / coco.py?

### ISSUE THREAD

**Issue by adeebakausar** [facebookresearch/maskrcnn-benchmark#1100]
## ❓ Questions and Help
from maskrcnn_benchmark.structures.bounding_box import BoxList

class MyDataset(object):
    def __init__(self, ...):
        # as you would do normally

    def __getitem__(self, idx):
        # load the image as a PIL Image
        image = ...

        # load the bounding boxes as a list of list of boxes
        # in this case, for illustrative purposes, we use
        # x1, y1, x2, y2 order.
        boxes = [[0, 0, 10, 10], [10, 20, 50, 50]]
        # and labels
        labels = torch.tensor([10, 20])

        # create a BoxList from the boxes
        boxlist = BoxList(boxes, image.size, mode="xyxy")
        # add the labels to the boxlist
        boxlist.add_field("labels", labels)

        if self.transforms:
            image, boxlist = self.transforms(image, boxlist)

        # return the image, the boxlist and the idx in your dataset
        return image, boxlist, idx

    def get_img_info(self, idx):
        # get img_height and img_width. This is used if
        # we want to split the batches according to the aspect ratio
        # of the image, as it can be more efficient than loading the
        # image from disk
        return {"height": img_height, "width": img_width}

**Comment by botcs** [comment]
Sorry, what's the question?

**Comment by Darshan2701** [comment]
@adeebakausar The question is very unclear, please make sure to add more clear information in order for the community to respond. :)

**Comment by adeebakausar** [comment]
Can you use any example to explain the following lines or can you upload
the coco.py file with a new customer dataset class?

This class has to be created in
maskrcnn-benchmark/maskrcnn_benchmark/data/datasets folder, same as
coco.py, and included in the init.py.
